apps/client/src/intros/shared/DynamicSTTRIntro.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { autofillMap } from "../../config/autofill";
+import { Intro } from ".";
+
+type Props = {
+  checker: any;
+};
+
+export default ({ checker }: Props) => {
+  const dependantOnQuestions = checker._getUpcomingQuestions().length > 0;
+  const dependantOnSituation = checker.getAutofillDataNeeds(autofillMap);
```

Comment:
Can you not send only these two instead of the whole checker?

Reply:
Yes we could ben then this file does nothing anymore. So you'd basically move this logic into IntroPage itself. That _is_ an option, but maybe IntroPage becomes a bit complicated handling all flows including dynamic + configured topics. What do you think?